src/worker/loan/arbiter/mailer.js

```diff
@@ -18,12 +18,20 @@ function defineMailerJobs (agenda) {
 
   agenda.define('mail-loan-cancelled', async (job, done) => {
     const { data } = job.attrs
-    const { emails } = data
+    const { emails, approved, collateralRequirementsMet, minCollateralAmount } = data
 
     const subject = 'Your loan has been cancelled.'
     const templateId = process.env.SENDGRID_LOAN_CANCELLED_TEMPLATE_ID
 
-    sendEmail(emails, subject, data, templateId)
+    let reason = `${minCollateralAmount} BTC was not locked as collateral within an hour and your loan request has been cancelled.`
```

Comment:
We might need to think about the messaging for this. Because it's technically within 2 hours. But it shows 1 hour on the interface.